packages/router/src/grpc/view-protocol-server/asset-metadata-by-id.test.ts

```diff
@@ -75,10 +75,11 @@ describe('AssetMetadataById request handler', () => {
     expect(metadataByIdResponse.denomMetadata?.equals(metadataFromNode)).toBeTruthy();
   });
 
-  test('should fail to get metadata when metadata not found in idb and node', async () => {
+  test('should return an empty response when metadata not found in idb and node', async () => {
```

Comment:
There's really no need to throw, since it's acceptable to return an empty response from this RPC method. So I changed the behavior.

Reply:
Response on other comment